--- vacuum-1.0.2/src/packages/linux/vacuum.desktop~	2010-01-18 06:49:05.000000000 +0100
+++ vacuum-1.0.2/src/packages/linux/vacuum.desktop	2011-02-22 16:26:13.691779398 +0100
@@ -2,10 +2,13 @@
 Encoding=UTF-8
 Name=Vacuum IM
 GenericName=Jabber Client
+GenericName[pl]=Klient Jabbera
 Comment=Communicate over the Jabber network
-Exec=vacuum
-Icon=vacuum
+Comment[pl]=Komunikacja poprzez sieć Jabber
+Exec=vacuum-im
+Icon=vacuum-im
 StartupNotify=true
 Type=Application
 Terminal=false
-Categories=Network;InstantMessaging;Qt
+Categories=Qt;Network;InstantMessaging;
+# vi: encoding=utf-8
