--- vacuum-im-r1503/src/packages/linux/vacuum.desktop~	2011-02-24 15:49:19.000000000 +0100
+++ vacuum-im-r1503/src/packages/linux/vacuum.desktop	2011-02-24 16:46:19.930943913 +0100
@@ -2,12 +2,15 @@
 Encoding=UTF-8
 Name=Vacuum-IM
 GenericName=Jabber Client
+GenericName[pl]=Klient Jabbera
 GenericName[ru]=Jabber клиент
 Comment=Communicate over the Jabber network
+Comment[pl]=Komunikacja poprzez sieć Jabber
 Comment[ru]=Общение в сети Jabber
-Exec=vacuum
-Icon=vacuum
+Exec=vacuum-im
+Icon=vacuum-im
 StartupNotify=true
 Type=Application
 Terminal=false
-Categories=Network;InstantMessaging;Qt;
+Categories=Qt;Network;InstantMessaging;
+# vi: encoding=utf-8
