--- vacuum-1.1.1/src/utils/utils.pro~	2011-08-19 19:42:49.000000000 +0200
+++ vacuum-1.1.1/src/utils/utils.pro	2011-09-06 09:36:02.872315019 +0200
@@ -18,7 +18,7 @@
 }
 
 LIBS              += -L../libs
-LIBS              += -lzlib -lidn -lminizip -lqxtglobalshortcut -lidle
+LIBS              += -lzlib -lidn -lminizip -lqxtglobalshortcut -lidle -lz
 macx {
   QMAKE_LFLAGS    += -framework Carbon
 } else:unix:!haiku {
